src/components/SiteTrackingRequired/SiteTrackingRequired.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { render, cleanup, waitForDomChange } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../services/pure-di';
+import { SiteTrackingRequired, SiteTrackingNotAvailableReasons } from './SiteTrackingRequired';
+
+describe('site tracking', () => {
+  afterEach(cleanup);
+
+  it('should show free account messages', () => {
+    // Arrange
+    const reason = SiteTrackingNotAvailableReasons.freeAccount;
+    const dependencies = {
+      dopplerLegacyClient: {
+        activateSiteTrackingTrial: async () => false,
+      },
+    };
+
+    // Act
+    const { getByText } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <DopplerIntlProvider>
+          <SiteTrackingRequired reason={reason} />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(getByText('reports.upgrade_account_free_title')).toBeInTheDocument();
+  });
+
+  it('should show trial not accepted messages', () => {
+    // Arrange
+    const reason = SiteTrackingNotAvailableReasons.trialNotAccepted;
+    const dependencies = {
+      dopplerLegacyClient: {
+        activateSiteTrackingTrial: async () => false,
```

Comment:
Since there is not _click_ in this test, are you sure that this setup is required?

Reply:
because use the dependencies in the element

Reply:
No, it is not necessary: 

![image](https://user-images.githubusercontent.com/1157864/62641640-6bc6f500-b91a-11e9-87f5-ad49b1b36e0b.png)

Reply:
![image](https://user-images.githubusercontent.com/1157864/62641706-9c0e9380-b91a-11e9-9d27-38e03085c044.png)

Reply:
wow when i tried it's fail, i check again

Reply:
done